study.md

```diff
@@ -34,7 +34,7 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+<!-- your answer here  :( )-->
```

Comment:
`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.